server/aws-lsp-codewhisperer/src/language-server/agenticChat/errors.ts

```diff
@@ -54,3 +54,12 @@ export function isInputTooLongError(error: unknown): boolean {
 
     return false
 }
+
+export function isImproperlyFormedRequest(error: unknown): boolean {
+    if (error instanceof Error) {
+        //  This is fragile (breaks if the backend changes their error message wording)
+        return error.message.includes('improperly formed request')
```

Comment:
`Improperly formed request` I think `.includes()` is case sensitive

Reply:
fixed it.